tox.ini

```diff
@@ -26,19 +24,26 @@ extras =
 commands =
   ruff format --check qiskit_addon_sqd/ docs/ test/
   ruff check qiskit_addon_sqd/ docs/ test/
+  ruff check --preview --select CPY001 --exclude "*.ipynb" qiskit_addon_sqd/ test/
   nbqa ruff docs/
+  pydocstyle qiskit_addon_sqd/
   mypy qiskit_addon_sqd/
-  #reno lint
   pylint -rn qiskit_addon_sqd/ test/
   nbqa pylint -rn docs/
-  pydocstyle qiskit_addon_sqd/
+  reno lint
```

Comment:
I went ahead and removed typos from the linter. It found one typo and one false positive and is now failing in CI and giving no indication why (I can't reproduce locally)

I feel fairly strongly that typos gets in the way more than it helps (Ive had trouble getting dictionaries to be read and trouble getting CI to tell me what happened with `typos`). Unless you feel strongly about this, I may leave it out of the linter

Reply:
It surprises me, that `typos` caused this trouble. Here are some notes:

- `typos` appears to never have been declared as a dependency (smth I overlooked as well) which makes me wonder why trying to execute the command gave an exit code `2` rather than `127`, but could be the reason why it kept failing at the end
- `typos` does *NOT* use a dictionary by design. Instead, it keeps an internal list of common typos which it detects and marks. This cuts down a *LOT* on false positives compared to e.g. pylint's spell checker. See [here](https://github.com/crate-ci/typos/blob/945d407a5fc9097f020969446a16f581612ab4df/docs/design.md) for more details.